### PR TITLE
feat: add svelte preset

### PR DESCRIPTION
**Pull request by userquin**
This PR includes svelte preset:
- svelte lifecycle, tick, context and event dispatcher: [svelte](https://svelte.dev/docs#svelte)
- [svelte/animate](https://svelte.dev/docs#svelte_animate)
- [svelte/easing](https://svelte.dev/docs#svelte_easing)
- [svelte/motion](https://svelte.dev/docs#svelte_motion)
- [svelte/store](https://svelte.dev/docs#svelte_store)
- [svelte/transition](https://svelte.dev/docs#svelte_transition)

@antfu I'll add some example for react and for svelte using vite on examples package/directory, when this PR merged.

### CODE REVIEW

**Comment by userquin** [comment]
The build on windows fails, if I remove the single quotes around `src/*.ts` just works, but I don't know it this change breaks `macosx and linux` builds)

**Comment by userquin** [comment]
I need to add also `linear` on easing...

File: src/presets/svelte-transition.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ImportsMap } from '../types'
+
+import { SvelteTransitionAPI } from './svelte'
+
+export default <ImportsMap>({
+  'svelte/transition': SvelteTransitionAPI,
```
Comment (antfu) [inline src/presets/svelte-transition.ts:6]: we could simply have them in one file

Comment (userquin) [reply src/presets/svelte-transition.ts:6]: so, remove all single entries?

Comment (userquin) [reply src/presets/svelte-transition.ts:6]: should I remove the animate, easing... entries from on the default export?

Comment (antfu) [reply src/presets/svelte-transition.ts:6]: yes, just have `presets/svelte` with named exports will do.

Comment (userquin) [reply src/presets/svelte-transition.ts]: done

**Pull request merged**